generate_build_config.py

```diff
@@ -51,6 +51,61 @@
 - chroot $CHROOT_ROOT apt-get -y update
 """  # noqa: E501
 
+SETUP_CONTENT = b"""\
+#!/bin/sh -eux
+mv /usr/sbin/grub-probe /usr/sbin/grub-probe.dist
+cat <<"PSUEDO_GRUB_PROBE" > /usr/sbin/grub-probe
+#!/bin/sh
+Usage() {
+   cat <<EOF
+Usage: euca-psuedo-grub-probe
```

Comment:
Or change it to echo out the options that were passed in to make it easier for us to debug issues when we only get a log file.